src/test/java/io/cdap/plugin/sfmc/sink/DataExtensionInfoTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.sfmc.sink;
+
+import com.exacttarget.fuelsdk.ETDataExtensionColumn;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DataExtensionInfoTest {
+
+  public static final String TEST_EXTERNALKEY = "test-externalKey";
+
+
```

Comment:
nit: remove empty line. please go over the PR and remove extra blank lines in other places

Reply:
removed